### ISSUE TITLE
Setup / Cleanup method for test when expect/where construction is used

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sampig/spock#255]
```
Setup/Cleanup methods is called for each data-row from where block. Is it 
possible to execute setup/cleanup method only once per test (not per data-row)
```

Original issue reported on code.google.com by `belov...@gmail.com` on 6 Jun 2012 at 7:17

**Comment by GoogleCodeExporter** [comment]
```
No, it isn't possible. If you'd like to discuss this, please use 
http://forum.spockframework.org
```

Original comment by `pnied...@gmail.com` on 6 Jun 2012 at 11:51
- Changed state: **WontFix**
- Added labels: **Milestone-0.7**
